dll/win32/msv1_0/msv1_0.h

```diff
@@ -357,4 +357,10 @@ SystemFunction012(const BYTE *in,
                   const BYTE *key,
                   LPBYTE out);
 
+VOID
+NTAPI
+RtlRunDecodeUnicodeString(
+    IN UCHAR Hash,
+    IN OUT PUNICODE_STRING String);
+
```

Comment:
@HBelusca: i know this is the wrong place. Any suggestion for the correct header? Is rtlfuncs.h a good place for it?

Reply:
Yes, somewhere around line 2437 of sdk/include/ndk/rtlfuncs.h . But this function (and its counterpart `RtlRunEncodeUnicodeString()`) must be declared within a `#ifdef NTOS_MODE_USER ... #endif` block.

Reply:
Should i add the counterpart RtlRunEncodeUnicodeString too? Or it is better to wait if someon needs it.

Reply:
Yes in the rtl header please add the Encode one too.

Reply:
done